### PR TITLE
Song components

### PR DESCRIPTION
**Pull request by vandyliu**
2021-06-10_23-46-37_chrome

Give me all your nits

My React and CSS is so so so so so so so so so so so so so bad, so that'll probably need to be tweaked

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +55,12 @@
...
 }
+
+/* DivTable.com */
+.divTable{ display: table; }
+.divTableRow { display: table-row; }
+.divTableHeading { display: table-header-group;}
+.divTableCell, .divTableHead { display: table-cell;}
+.divTableHeading { display: table-header-group;}
+.divTableFoot { display: table-footer-group;}
+.divTableBody { display: table-row-group;}
```
Comment (ianmah) [inline src/App.css:66]: are u still using these

File: src/components/SongHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const ImgDiv = styled.div`
+  display: table-cell;
+  vertical-align: middle;
+  font-size: 18px;
+  font-weight: bolder;
+  width: 4%;
+  padding: 0px 0px 0px 0px;
```
Comment (ianmah) [inline src/components/SongHeader.jsx:10]:
```suggestion
  padding: 0;
```

optional

File: src/components/SongList.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+function SongList(props) {
+    // const [pic, setPic] = useState(props.pic);
+
+    return (
+        <SongDiv>
+            <Table>
+                <SongHeader />
+                {songs.map((s, i) => (
+                    <Song {...s.track} />
```
Comment (ianmah) [inline src/components/SongList.jsx:36]:
```suggestion
                {songs.map((song, i) => (
                    <Song {...song.track} />
```
nit aha

**Review by ianmah — APPROVED** [review]
Woo wee!

File: src/App.css (hunk 1 above)
Comment (vandyliu) [reply src/App.css]: oops removed

**Pull request merged**